src/constants/templates.js

```diff
@@ -1,11 +1,10 @@
 const templates = [
   {
-    value: 'general',
-    label: 'General',
+    value: 'cryptocurency-transaction',
+    label: 'Cryptocurrency Transaction',
     content:
-      'This agreement is made effective as of [[Effective Date]] between [[Party A]] and [[Party B]].\n\n\nOffering\n--------\n\n[[Specify the Offering here and be precise in describing deliverables.]]\n\nThe period over which this agreement applies is as follows:\nDelivery: [[Over what period will the offering be made available?]]\nAuthority: [[Over what period will the authority be bestowed?]]\n\n\nDispute Resolution\n--------\n\n[[Party A]] and [[Party B]] will attempt to resolve any dispute arising through friendly negotiation (partial payments/mediation).\nIf the matter is not resolved, parties agree to submit to arbitration in Kleros.'
-  },
-  { value: 'custom', label: 'Custom', content: '' }
+      'The purpose of this contract is to provide clear guidelines for [Seller] and [Buyer] of [cryptoasset] in order to facilitate the transfer of said asset. \n\n[Buyer] deposits funds into the escrow.\n\n[Seller] will send [cryptoasset] into [address] within [due date].\n\nOnce [Buyer] receives [cryptoasset] they release funds to seller.\n\nIf [seller] does not release [cryptoasset] before contract end date and no dispute is brought, the contract automatically sends [cryptoasset] to [buyer]'
```

Comment:
This is outdated. See https://docs.google.com/document/d/1LzDgLpXIU81byNi2A2bBb1_kL9Km7jVFrAPMcUPWObA/edit#heading=h.77zcrpw9i7y3

Reply:
So there is no contract text at all then. We should remove that whole text box

Reply:
It becomes the meta evidence `description`. We need it there.